12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1,6 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+
+
+import sys
+
+letters = [chr(i) for i in range(97,123)]

```

Comment:
Better to use `ord('a')` and `ord('z')+1` as limits as it results in more readable code